service/worker/deletenamespace/deleteexecutions/activities.go

```diff
@@ -180,7 +188,15 @@ func (a *Activities) DeleteExecutionsActivity(ctx context.Context, params Delete
 		logger.Error("Unable to list all workflow executions.", tag.Error(err))
 		return result, err
 	}
-	rateLimiter := quotas.NewRateLimiter(float64(params.RPS), params.RPS)
+	var rateLimiter *quotas.RateLimiterImpl
+	deleteRPS, cancelDeleteRPSWatch := a.deleteActivityRPS(func(newRPS int) {
+		if rateLimiter != nil {
+			rateLimiter.SetRPS(float64(newRPS))
```

Comment:
Thanks to ability to subscribe to DC changes (@dnr) and change RPS on the fly (@wxing1292) this PR is straight forward.